Userland/Applications/SoundPlayer/PlaylistWidget.h

```diff
@@ -40,19 +40,22 @@ class PlaylistTableView : public GUI::TableView {
     void doubleclick_event(GUI::MouseEvent& event) override;
 
     Function<void(const Gfx::Point<int>&)> on_doubleclick;
+
+private:
+    PlaylistTableView() = default;
```

Comment:
Move to implementation file; this creates 3 extra symbol references in object files including this header.